onnx/defs/reduction/defs.cc

```diff
@@ -78,6 +82,15 @@ False instead of True.)DOC";
           AttributeProto::INTS,
           OPTIONAL_VALUE);
     }
+    if (complement_axis) {
+      schema.Attr(
+          "complement_axis",
+          "Whether to reduce axes complementarily, "
+          "default is False (reduce target axis). "
+          "This attribute is used in conjunction with axis attribute.",
```

Comment:
in conjunction with "axes" attribute. In context of reduction ops axes is plural so use the axes instead of axis